### PR TITLE
FSE: Update to 0.21

### PR DESCRIPTION
**Pull request by noahtallen**
### Changes proposed in this Pull Request
Since the latest FSE update on dotcom (D39601-code) didn't bump the constant versions of the plugin, we need to:
1. Get those versions on .com
2. Make sure the code on .com includes the latest changes in FSE (since it is possible the update happened with code on a stale branch).

### Testing instructions
1. Test the attached diff
2. Deploy to dotcom
3. Deploy to dotorg
4 Merge this PR

### CODE REVIEW

File: client/lib/analytics/ad-tracking.js [hunk 1]
```diff
@@ -587,13 +587,9 @@ export function trackCustomAdWordsRemarketingEvent( properties ) {
...
 
 function splitWpcomJetpackCartInfo( cart ) {
-	// Note: reduce() requires a non 0-length array.
-	const jetpackCost =
-		0 === cart.products.length
-			? 0
-			: cart.products
-					.map( product => ( productsValues.isJetpackPlan( product ) ? product.cost : 0 ) )
-					.reduce( ( accumulator, cost ) => accumulator + cost );
+	const jetpackCost = cart.products
```
Comment (vindl) [inline client/lib/analytics/ad-tracking.js]: Why are these changes in the version bump PR?

File: client/my-sites/migrate/section-migrate.jsx [hunk 2]
```diff
@@ -365,28 +365,29 @@ class SectionMigrate extends Component {
 					{ translate( 'There was an error with your import.' ) }
 					<br />
 					{ this.state.errorMessage }
+
+					<p className="migrate__info">
+						{ translate(
+							'{{supportLink}}Contact us{{/supportLink}} so we can' +
```
Comment (a8ci18n) [inline client/my-sites/migrate/section-migrate.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **14** times:
`translate( 'Or {{supportLink}}contact us{{/supportLink}} so we can figure out exactly what needs adjusting and get your site imported.' )` *ES Score: 10*

File: client/lib/analytics/ad-tracking.js (hunk 1 above)
Comment (noahtallen) [reply client/lib/analytics/ad-tracking.js]: @vindl We had to rebase the branch to delete a commit introduced by SPT.

Comment (noahtallen) [reply client/lib/analytics/ad-tracking.js]: I did `git rebase -i` to the commit right before the one introducing the SPT issue. And then deleted the SPT commit that is problematic. Then force pushed, which included those changes. Is there a better way to do it?

**Comment by noahtallen** [comment]
I reset this to master since removing the commit did not fix the issue.

**Comment by noahtallen** [comment]
Everything is now deployed!

**Pull request merged**